/scripts/regression-net-socket.js:7
  throw new Error('trace');
  ^

Error: trace
    at Server.<anonymous> (/scripts/regression-net-socket.js:7:9)
    at emitOne (events.js:96:13)
    at Server.emit (events.js:188:7)
    at TCP.onconnection (net.js:1460:8)
    at createServerHandle (net.js:1181:14)
    at Server._listen2 (net.js:1238:14)
    at listen (net.js:1290:10)
    at net.js:1400:9
    at GetAddrInfoReqWrap.asyncCallback [as callback] (dns.js:65:16)
    at GetAddrInfoReqWrap.onlookup [as oncomplete] (dns.js:84:10)
    at Object.lookup (dns.js:167:19)
    at listenAfterLookup (net.js:1395:20)
    at Server.listen (net.js:1391:5)
    at Object.<anonymous> (/scripts/regression-net-socket.js:10:8)
    at Module._compile (module.js:541:32)
    at Object.Module._extensions..js (module.js:550:10)
    at Module.load (module.js:458:32)
    at tryModuleLoad (module.js:417:12)
    at Function.Module._load (module.js:409:3)
    at Module.runMain (module.js:575:10)
    at run (bootstrap_node.js:340:7)
    at startup (bootstrap_node.js:132:9)
    at bootstrap_node.js:455:3
